release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -57,6 +57,18 @@ This release adds improvements related to the following components and concepts.
 
 [id="ocp-4-14-web-console"]
 === Web console
+
+[id="ocp-4-14-openshift-cli"]
+=== OpenShift CLI (oc)
+
+.oc-mirror supporting multi-arch OCI local images for catalogs
+
+With {product-title} {product-version}, oc-mirror supports multi-arch OCI local images for catalogs.
+
+OCI layouts consist of an index.json file that identifies the images held within them on disk. This index.json file can reference any number of single or multi architecture images. However, oc mirror only references a single image at a time in a given OCI layout. The image stored in the OCI layout can be either a single architecture image (i.e. an image manifest) or a multi-architecture image (i.e. a manifest list).
+
+The ImageSetConfiguration stores the OCI images, and after processing the catalog, the catalog content adds new layers representing the content of all images in the layout. The ImageBuilder is modified to handle image updates for both single-arch and multiarch images.
```

Comment:
better to change to "single-arch and multi-arch images"

Reply:
Ack.

Reply:
@subhtk we still see 'multiarch' here , better to use `multi-architecture`